mavros_extras/src/plugins/distance_sensor.cpp

```diff
@@ -191,10 +191,12 @@ class DistanceSensorPlugin : public MavRosPlugin {
 			return;
 		}
 
-		if (sensor->orientation >= 0 && dist_sen.orientation != sensor->orientation) {
+		if (UAS::orientation_from_str(sensor->orientation) >= 0 && dist_sen.orientation != UAS::orientation_from_str(sensor->orientation)) {
```

Comment:
...
This still is a int comparison... The only thing I can do is store the value previously before do the comparison... Didn't broke nothing